pytubefix/contrib/search.py

```diff
@@ -14,7 +14,8 @@
 class Search:
     def __init__(
             self, query: str,
-            client: str = InnerTube().client_name,
+            params: Optional[Dict[str, str]] = None,
+            client: str = 'WEB',
```

Comment:
I think it would be more intuitive for the user to leave the client as the second parameter.